music21/setup.py

```diff
@@ -0,0 +1,75 @@
+##!/usr/bin/env python
```

Comment:
This file needs to be deleted -- there should not be a setup.py in the music21 directory, only in music21base.